samples/AzureStorageQueuesPubSubSample/AzureStorageQueuesPubSub.cs

```diff
@@ -0,0 +1,141 @@
+using Azure.Storage.Queues;
+using Dapr.PluggableComponents.Components;
+using Dapr.PluggableComponents.Components.PubSub;
+
+internal sealed class AzureStorageQueuesPubSub : IPubSub
+{
+    private readonly ILogger<AzureStorageQueuesPubSub> logger;
+
+    private string? connectionString;
+    private string? queueName;
+    private TimeSpan pollInterval = TimeSpan.FromSeconds(5);
+    private int maxMessages = 5; 
+
+    public AzureStorageQueuesPubSub(ILogger<AzureStorageQueuesPubSub> logger)
+    {
+        this.logger = logger ?? throw new ArgumentNullException(nameof(logger));
+    }
+
+    #region IPubSub Members
+
+    public Task InitAsync(MetadataRequest request, CancellationToken cancellationToken = default)
+    {
+        this.connectionString = request.Properties["connectionString"];
+        this.queueName = request.Properties["queueName"];
+
+        if (request.Properties.TryGetValue("pollInterval", out var pollIntervalString))
+        {
+            this.pollInterval = TimeSpan.FromSeconds(Int32.Parse(pollIntervalString));
+        }
+
+        if (request.Properties.TryGetValue("maxMessages", out var maxMessagesString))
+        {
+            this.maxMessages = Int32.Parse(maxMessagesString);
+        }
+
+        return Task.CompletedTask;
+    }
+
+    public async Task PublishAsync(PubSubPublishRequest request, CancellationToken cancellationToken = default)
+    {
+        var queueClient = new QueueClient(this.connectionString, this.queueName);
```

Comment:
wondering if you can just create once on the Initialization and reuse the instance for both methods.

qq: does this queue has a dispose method?

Reply:
`QueueClient` isn't `IDisposable` nor (based on docs and a brief perusal of the code) particularly stateful, so I don't think it's critical to share across method calls.  As a sample, I'm trying to balance simplicity,  readability, and optimization.